ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -67,6 +68,7 @@
   private long clusterWiseAllocatedUsableCapacityBytes;
   private final HelixClusterManagerCallback helixClusterManagerCallback;
   private final AtomicReference<Exception> initializationException = new AtomicReference<>();
+  private final AtomicLong sealedStateChangeCounter = new AtomicLong(0);
```

Comment:
Yes - this can get updated in the context of InstanceConfigChange notifications (that can come in parallel from different ZK servers).